CreeDictionary/API/search/ranking_test.py

```diff
@@ -0,0 +1,45 @@
+import pytest
```

Comment:
Could you rename this file to `test_ranking.py`?

I dislike our current practice of dumping all of our tests in `/CreeDictionary/tests`, so you don't have to do that, but I'd appreciate it if all unit tests began with the string `test_` — that way, it's easy to scan in a sorted directory listing where the tests are, especially if there not all in one big (strangley placed) directory structure!